sematic/resolvers/worker.py

```diff
@@ -169,12 +175,24 @@ def main(run_id: str, resolve: bool):
         raise e
 
 
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO)
+def _create_log_file_path(file_name: str) -> str:
+    temp_dir = tempfile.gettempdir()
+    sematic_temp_logs_dir = pathlib.Path(temp_dir) / pathlib.Path("sematic_logs")
+    if not pathlib.Path(sematic_temp_logs_dir).exists():
+        os.mkdir(sematic_temp_logs_dir)
+    return (pathlib.Path(sematic_temp_logs_dir) / pathlib.Path(file_name)).as_posix()
 
+
+if __name__ == "__main__":
+    print("Starting Sematic Worker")
     args = parse_args()
+    log_kind = "resolve" if args.resolve else "calculation"
+    log_prefix = f"logs/run_id/{args.run_id}/{log_kind}"
```

Comment:
When you see a UUID it's not always obvious what it is. Making it `run_id/<the id>` makes it obvious what id you're looking at. Same reason you might use a `NamedTuple` rather than a `Tuple`. I don't believe it should cost anything extra to have, performance or money-wise.